packages/atlas/src/components/NftTile/NftTileDetails.tsx

```diff
@@ -35,22 +36,24 @@ export type NftTileDetailsProps = {
   owner?: Member
   creator?: Member
   role?: 'owner' | 'viewer'
-  auction?: 'none' | 'minBid' | 'topBid' | 'waiting'
+  nftState?: 'idle' | 'on-sale' | 'auction'
   buyNowPrice?: number | null
-  minBid?: number | null
+  startingPrice?: number | null
   topBid?: number | null
   title?: string | null
   hovered?: boolean
+  interactable?: boolean
+  videoHref?: string
 }
 
 type TileSize = 'small' | 'medium'
 
-type DetailsContent = {
+type DetailsContentProps = {
   caption: string
   icon: React.ReactNode
   content: string | number
   secondary?: boolean
-  tileSize?: TileSize
+  tileSize: TileSize | undefined
```

Comment:
```suggestion
  tileSize?: TileSize
```

Reply:
no, I want this to be passed always but I also allow it to be undefined